dist/src/main/dist/conf/benchmark-report.py

```diff
@@ -1147,7 +1147,9 @@ def generate(self):
         with open(file_name, 'w') as f:
             f.write(self.report)
 
-        print("HTML report generated at: " + file_name)
+        # print a clickable link to report file
+        print("HTML report generated at: "+"\u001b]8;;file://"
+              + file_name + "\u001b\\" + file_name + "\u001b]8;;\u001b\\")
```

Comment:
In my Ubuntu Mate, this way helps neither in the MATE Terminal nor in the Terminator. :( Nevertheless, if we keep the scheme prefix in the path, it will work OOTB.

```suggestion
        file_url = "file://" + file_name
        print(f"HTML report generated at: \u001b]8;;{file_url}\u001b\\{file_url}\u001b]8;;\u001b\\")
```

Reply:
updated.